libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -560,11 +564,11 @@ public void onMediaUploadSucceeded(final String localMediaId, final MediaFile me
         private final String mId;
         private final String mAttributeName;
 
-        static ImagePredicate localMediaIdPredicate(String id) {
+        private static ImagePredicate localMediaIdPredicate(String id) {
```

Comment:
It doesn't look like this breaks anything, but since we didn't actually need to change the scope of these methods it might be better to leave them package-private as they originally were.